pkg/artifactory/resource/repository/remote/resource_artifactory_remote_terraform_repository.go

```diff
@@ -10,36 +10,42 @@ import (
 
 type TerraformRemoteRepo struct {
 	RepositoryRemoteBaseParams
-	TerraformRegistryUrl  string `hcl:"terraform_registry_url" json:"terraformRegistryUrl"`
-	TerraformProvidersUrl string `hcl:"terraform_providers_url" json:"terraformProvidersUrl"`
+	TerraformRegistryUrl  string `json:"terraformRegistryUrl"`
+	TerraformProvidersUrl string `json:"terraformProvidersUrl"`
```

Comment:
This is just a question, what does changing this from HCL to json do?

Reply:
We are removing the redundant `hcl` tags not changing them to `json`.

Reply:
I realize now the hcl was removed, I guess my new question is why do we remove this?